internal/nodes/avi_model_l7_translator.go

```diff
@@ -218,7 +215,7 @@ func (o *AviObjectGraph) BuildTlsCertNode(svcLister *objects.SvcLister, tlsNode
 			return false
 		}
 	} else {
-		secretObj, err := mClient.CoreV1().Secrets(secretNS).Get(context.TODO(), secretName, metav1.GetOptions{})
+		secretObj, err := utils.GetInformers().SecretInformer.Lister().Secrets(namespace).Get(secretName)
```

Comment:
Should we use "secretNS" instead of "namespace" ?

Reply:
Done.